### PR TITLE
Fix: update the docs and auto function module with update of theme package.

### PR DESCRIPTION
**Pull request by Revathyvenugopal162**
- [ ] update the docs 
- [ ] auto function module with update of ansys-sphinx-theme package.
- [ ] update ansys-sphinx-theme

### CODE REVIEW

**Review by jorgepiloto — APPROVED** [review]
Good catch, @Revathyvenugopal162. Thanks for adding this.

**Comment by jorgepiloto** [comment]
Linking here https://github.com/ansys/ansys-sphinx-theme/pull/212.

**Comment by jorgepiloto** [comment]
I just debug this a bit more and found the issue. We are not using the multi-version switcher in this repository since we continuously deploy. However, this is causing some undesired behavior.

Run the doc build increasing the verbosity "-vv". You will see that the build fails close to some "version_switcher" option. Activating it allowed to build to run.

Let us investigate more about this, @Revathyvenugopal162.

File: requirements/requirements_doc.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 Sphinx==5.3.0
-ansys-sphinx-theme==0.8.2
+ansys-sphinx-theme==0.9.1
```
Comment (RobPasMue) [inline requirements/requirements_doc.txt:2]:
```suggestion
ansys-sphinx-theme==0.9.2
```
New version available... should it solve the problems?

**Review by RobPasMue — APPROVED** [review]
LGTM but we can wait until the problems are fixed

File: requirements/requirements_doc.txt (hunk 1 above)
Comment (jorgepiloto) [reply requirements/requirements_doc.txt]: Sadly, no. I had to perform that hot-fix after realizing that there was an issue with the typing. It was very werid.

Comment (jorgepiloto) [reply requirements/requirements_doc.txt]: But we need to apply that version for sure.

**Comment by Revathyvenugopal162** [comment]
> I just debug this a bit more and found the issue. We are not using the multi-version switcher in this repository since we continuously deploy. However, this is causing some undesired behavior.
> 
> Run the doc build increasing the verbosity "-vv". You will see that the build fails close to some "version_switcher" option. Activating it allowed to build to run.
> 
> Let us investigate more about this, @Revathyvenugopal162.

Linking here https://github.com/ansys/ansys-sphinx-theme/pull/214

File: requirements/requirements_doc.txt [hunk 2]
```diff
@@ -1,5 +1,5 @@
 Sphinx==5.3.0
-ansys-sphinx-theme==0.8.2
+ansys-sphinx-theme==0.9.2
```
Comment (Revathyvenugopal162) [inline requirements/requirements_doc.txt:2]:
```suggestion
ansys-sphinx-theme==0.9.3
```

**Pull request merged**